lib/utils/class-alerts.php

```diff
@@ -246,6 +246,7 @@ public function send_to_opsgenie( $message, $details, $kind = '', $interval = 1
 		$details[ 'message' ] = $message;
 
 		$body = [
+			'message'   => $message,
```

Comment:
It seems that the `message` thing is an artifact of my testing (bad copy-pasta), so it's not needed here.